src/store/chat/saga.ts

```diff
@@ -153,6 +148,7 @@ export function* performValidateActiveConversation(activeConversationId: string)
 
   let conversationId = activeConversationId;
   if (isAlias(activeConversationId)) {
+    activeConversationId = parseAlias(activeConversationId);
```

Comment:
this will just prefix the id with `#` before proceeding, so everything else (save to store, calling api etc) remains the same